src/main/resources/assets/traverse/lang/fr_fr.json

```diff
@@ -0,0 +1,59 @@
+{
+  "itemGroup.traverse.items": "Objets de Traverse",
+
+  "biome.traverse.arid_highlands": "Hautes terres arides",
+  "biome.traverse.autumnal_wooded_hills": "Collines automnales",
+  "biome.traverse.autumnal_woods": "Bois automnaux",
+  "biome.traverse.cliffs": "Falaises",
+  "biome.traverse.coniferous_forest": "Forêt de conifères",
+  "biome.traverse.coniferous_wooded_hills": "Collines de conifères",
+  "biome.traverse.desert_shrubland": "Désert buissonné",
+  "biome.traverse.high_coniferous_forest": "Forêt de grands conifères",
```

Comment:
The high coniferous forest doesn't have different trees, but it's actually just at a higher altitude. You could perhaps describe it as being more mountainous.